components/net/fetch/methods.rs

```diff
@@ -88,6 +89,11 @@ pub type DoneChannel = Option<(Sender<Data>, Receiver<Data>)>;
 
 /// [Fetch](https://fetch.spec.whatwg.org#concept-fetch)
 pub fn fetch(request: &mut Request, target: Target, context: &FetchContext) {
+    context
```

Comment:
Let's add a comment here like `// Step 7 of https://w3c.github.io/resource-timing/#processing-model`